subsys/bluetooth/audio/mpl.c

```diff
@@ -1046,14 +1046,22 @@ static void do_next_track(struct mpl_mediaplayer *pl)
 	}
 }
 
-static void do_first_track(struct mpl_mediaplayer *pl)
+static void do_first_track(struct mpl_mediaplayer *pl, bool group_change)
 {
+	bool track_changed;
```

Comment:
```suggestion
	bool track_changed = false;
```

Reply:
Yes, this fixes both PTS issue and [TS Errata](https://bluetooth.atlassian.net/browse/ES-24634)